src/templates/typescript/indexTemplate.ts

```diff
@@ -0,0 +1,7 @@
+import Handlebars from "./registerPartials";
+
+export const source = `
+export * from "./internal";
+`;
+
+export default Handlebars.compile<void>(source);
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5923420)